packages/toolpad-components/src/PageRow.tsx

```diff
@@ -30,7 +30,7 @@ export default createComponent(PageRow, {
   argTypes: {
     spacing: {
       typeDef: { type: 'number' },
-      defaultValue: 0,
+      defaultValue: 2,
```

Comment:
I think that we should aim for dense UIs, I would propose instead:

```suggestion
      defaultValue: 1,
```

Reply:
Agreed. Also, since at the moment this adds both margin and gap to the rows, it creates a gap of `2` when two rows stack. We should probably make it so that this only controls `gap`, and provide a similar thing for columns so that every gap between elements has the same size throughout by default.